### PR TITLE
fix(tests): add test to verify noImplicityAny builds work

### LINKED ISSUE #107: consider turning on noImplicitAny by default
currently our blueprints don't pass the noImplicitAny check - we should fix that.

- [ ] understand the impact
- [ ] change the [config](https://github.com/angular/angular-cli/blob/d89d717358cbecdd74244189234eb8740f12faa2/addon/ng2/blueprints/ng2/files/src/tsconfig.json#L10)
- [ ] update blueprints that fail the check

Current failures for an empty project:

```
[DiffingTSCompiler]: Typescript found the following errors:
  app/sample-45.spec.ts (7, 71): Parameter 'app' implicitly has an 'any' type.
  app/sample-45.spec.ts (12, 65): Parameter 'app' implicitly has an 'any' type.
  app/sample-45.ts (14, 17): Parameter 'meaning' implicitly has an 'any' type.
```

@rkirov @alexeagle - what's your guidance on this?

### PR DESCRIPTION
**Pull request by Brocco**
Resolved noImplicitAny issues in blueprints
Updated expect for ng build verifications
Fixes #107

### CODE REVIEW

**Comment by filipesilva** [comment]
lgtm, great job Mike!

File: addon/ng2/blueprints/pipe/files/src/app/pipes/__name__/__name__.ts [hunk 1]
```diff
@@ -6,7 +6,7 @@ import {Pipe, PipeTransform} from 'angular2/core';
 })
 export class <%= classifiedModuleName %> implements PipeTransform {
 
-  transform(value, args?) {
+  transform(value: any, args?: any): any {
```
Comment (gkalpak) [inline addon/ng2/blueprints/pipe/files/src/app/pipes/__name__/__name__.ts]: Shouldn't this be `args?: any[]` ?

File: tests/e2e/e2e_workflow.spec.js [hunk 2]
```diff
@@ -174,8 +174,37 @@ describe('Basic end-to-end Workflow', function () {
...
+    fs.readFile(configFilePath, 'utf8', function(err, data){
+      
+      var config = JSON.parse(data);
+      config.compilerOptions.noImplicitAny = true;
+      
+      fs.writeFile(configFilePath, JSON.stringify(config), function(){
+        //clear the dist folder
+        sh.rm('-rf', path.join(process.cwd(), 'dist'));
+        
+        return ng([
```
Comment (gkalpak) [inline tests/e2e/e2e_workflow.spec.js]: Is `return` needed here ?

Comment (Brocco) [inline tests/e2e/e2e_workflow.spec.js]: No, not needed. It was a result of copying in the promise implementation from above. But at the same time, right now that does not hurt anything. The next change I make in there, I'll clean that up, but I don't think it's worth a separate PR to remove it with no other changes.

**Pull request merged**